src/server/api/followers.ts

```diff
@@ -23,11 +23,14 @@ export const followerRoutes = (cfg: APIConfig, store: Store, apsystem: ActivityP
   }, async (request, reply) => {
     const { actor } = request.params
     const allowed = await apsystem.hasPermissionActorRequest(actor, request)
+    const collection = await apsystem.followersCollection(actor)
+
+    // Allow seeing follower count but not followers
+    // TODO: Allow list to be public?
     if (!allowed) {
-      return await reply.code(403).send('Not Allowed')
+      delete collection.items
```

Comment:
if i understand correctly, the followers are still webfingered but not shown publicly, right?

Reply:
In this case yes. I suppose I could just show the count though?

Reply:
I think it'll be faster and we can avoid launching lots of webfinger requests just to get an accurate count